README.md

```diff
@@ -17,10 +17,42 @@ There you'll find documentation, examples, announcements and more.
 
 ### Configuration
 
-Feedjira configuration options can be set by using a block
+#### Stripping whitespace from XML
+
+Feedjira can be configured to strip all whitespace but defaults to lstrip only:
 
 ```ruby
 Feedjira.configure do |config|
   config.strip_whitespace = true
 end
 ```
+
+#### Follow redirect limit
+
+For fetching feeds, the follow redirect limit defaults to 3 but can be set:
+
+```ruby
+Feedjira.configure do |config|
+  config.follow_redirect_limit = 5
+end
+```
+
+#### Request timeout
```

Comment:
What if we started with what's here and change the docs if needed. A lot of how this is documented is heavily borrowed from Thoughtbots gems.